index.rst

```diff
@@ -41,7 +43,7 @@ La Serena facilities has been divided in the following areas:
 - UPS room
 - Cooling room
 
-This document will focus on the first 3 areas
+This documentation will centralize its focus on the first 3 areas.
```

Comment:
"This documentation will focus in the first 3 areas." :)

Reply:
I actually thought of putting that too, I will revert the change. Thanks for the feedback Heinrich.